core/pta/scmi.c

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2019-2021, Linaro Limited
+ * Copyright (c) 2019-2021, STMicroelectronics
+ */
+#include <compiler.h>
+#include <config.h>
+#include <confine_array_index.h>
+#include <drivers/scmi-msg.h>
+#include <kernel/pseudo_ta.h>
+#include <pta_scmi_client.h>
+#include <stdint.h>
+#include <string.h>
+
+static TEE_Result cmd_capabilities(void *session __unused, uint32_t ptypes,
```

Comment:
Are you planning to use the `session` parameter in any of these functions? If not, why not remove them?